bootstrap/GetToken.py

```diff
@@ -0,0 +1,83 @@
+import sys
+import json
+import datetime
+import jwt
+
+from Helper import *
+
+if sys.version_info[0] == 2:
+    from urllib import urlencode
+if sys.version_info[0] >= 3:
+    from urllib.parse import urlencode
+
+logger = setup_logger(__name__)
+
+
+def get_jwt_token(ims_host, org_id, tech_acct, api_key, priv_key):
```

Comment:
If `get_jwt_token` is only used from within other methods in this file, should it be marked as less public (somehow)?